docs/UserGuide.md

```diff
@@ -19,11 +19,10 @@ more effectively than traditional GUI apps.
         * Adding help tag to a student: [`helpstu`](#adding-help-tag-helpstu)
         * Deleting a student: [`deletestu`](#deleting-a-student-deletestu)
         * Finding a student: [`findstu`](#finding-a-student-findstu)
-        * Listing all students: [`liststu`](#listing-all-students-liststu)
```

Comment:
If I am not wrong after `findstu`, the list will only show a certain number of students, `liststu` will then list all the students again so it is still valid.

Reply:
Added back in